csrc/custom_all_reduce_test.cu

```diff
@@ -330,10 +330,17 @@ int main(int argc, char** argv) {
   //     run<half>(myRank, nRanks, comm, threads, block_limit, 4096 * 1024);
   //   }
   // }
+#ifdef USE _ROCM
```

Comment:
I understand that #137 fixed this typo, but this should cause vLLM compilation to fail. How was this PR merged when it doesn't even compile?